recipe/meta.yaml

```diff
@@ -20,31 +20,45 @@ source:
 
 build:
   number: 0
+  # only currently needed for pytorch on linux-64. Remove these when building the whole llvm suite
+  skip: True  # [not (linux and x86_64)]
 
 requirements:
   build:
     - cmake
     - ninja
-    - {{ stdlib('c') }}
+    #- {{ stdlib('c') }}
     - {{ compiler('cxx') }}
-    - llvmdev {{ version }}  # [build_platform != target_platform]
+    - patch       # [not win]
+    - m2-patch    # [win]
+    # both of these required to run testing
+    - lit     {{ version }}
+    - llvmdev {{ version }}
+    - python
```

Comment:
Do we need a `ignore_run_exports` on python?

Reply:
not if it's in build

Reply:
I removed the one in host, too